src/Microsoft.Azure.WebJobs.Host/Blobs/Listeners/SharedBlobListener.cs

```diff
@@ -91,14 +91,8 @@ public void Dispose()
 
         private static IBlobListenerStrategy CreateStrategy(IStorageAccount account)
         {
-            if (!StorageClient.IsDevelopmentStorageAccount(account))
-            {
-                return new PollLogsStrategy();
-            }
-            else
-            {
-                return new ScanContainersStrategy();
-            }
+            account.ToString();
```

Comment:
that should be removed, the parameter should be removed all together, I hacked it quickly during testing forgot to remove it